mmv1/products/compute/Instance.yaml

```diff
@@ -487,6 +487,13 @@ properties:
             should be specified.
         # networkInterfaces.kind is not necessary for convergence.
           custom_expand: 'templates/terraform/custom_expand/resourceref_with_validation.go.erb'
+        - !ruby/object:Api::Type::ResourceRef
+          name: 'networkAttachment'
+          resource: 'networkAttachment'
+          imports: 'selfLink'
+          description: |
+            The URL of the network attachment that this interface should connect to in the following format:
+            projects/{projectNumber}/regions/{region_name}/networkAttachments/{network_attachment_name}.
```

Comment:
@c2thorn could you please, help me with this resource. I can't see this change reflected on the downstream repo! Do you have in mind why?

Reply:
@diogoEsteves42 Compute instance is not managed by this file due to the `exclude_resource` attribute., this is essentially left as a stub for some other functionality. Compute Instance is one of our most complicated resources with enough custom behavior that we have kept it handwritten in https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/services/compute/resource_compute_instance.go.erb